guide/payments/request.md

```diff
@@ -82,31 +70,35 @@ Below is an overview of these various entry points:
 
 The most common way bitcoin is requested today is in the form of single-use payment requests. For Lightning, a standard [BOLT11](https://github.com/lightning/bolts/blob/master/11-payment-encoding.md) invoice is used. For on-chain, [addresses of various types]({{ "/guide/glossary/address/" | relative_url }}) are used. Addresses can be reused though this is not advised due to [privacy concerns]({{ "/guide/payments/privacy/#generate-a-new-address-for-each-payment" | relative_url }}). Single-use invoices and addresses need to be easily and conveniently generated within a wallet application.
 
-### Amount
+As single-use payment requests are convenient for one-time or in-person payments, generating them should be possible from your applications home screen in a fast, convenient, and, private manner.
 
-When generating a single-use payment request, the receiver may want to add an amount to be paid. Entering an amount from the home screen can speed up the requesting flow. Alternatively, users can add an amount to an already generated payment request. Entering an amount needs to be clear to prevent users entering incorrect amounts.
+Do not show balances on your home screen when generating single-use payment requests as a third party, especially if requesting in person, is likely to see the users balance which is a privacy concern. Home screens can also enter users straight into a request flow, such as entering an amount to be requested, speeding up the process.
 
-Amount units need to be toggleable between bitcoin, satoshi and local currency values as senders may request in a set unit. Ensure the unit being entered is prominent to prevent entering of the wrong unit amount. More on the [Units & Symbols page]({{ "/guide/payments/units-and-symbols/" | relative_url }}).
+### Amounts
 
-### Address types
+When generating single-use payment requests, the receiver may want to add an amount to be paid.
 
-There are several address types that can be used for on-chain payment requests. Each address type has unique benefits, such as SegWit saving on network fees. It's recommended to encourage users to use the latest - more feature rich address types. However, older address types should be supported for seamless backwards compatibility. More on the [address types]({{ "/guide/glossary/address/" | relative_url }}) page.
+As mentioned above, entering an amount from the home screen can speed up the requesting flow and is also more private. It should also be possible for users to tap request and generate a zero amount payment request if they want the sender to set the amount.
 
-### Meta data
+Zero amount invoices also limit both sender and receivers exposure to price volatility, which can be a major usability issue in places using bitcoin as a means of payment like El Salvador where they are denominating their payments in fiat.
```

Comment:
> like El Salvador where they are denominating their payments in fiat

Can we make a blanket statement like that? I'd just say that this is helpful for people who primarily deal with fiat.

Reply:
> Zero amount invoices also limit both sender and receivers exposure to price volatility

How does it limit exposure? We should explain this.